lib/SILGen/SILGenConstructor.cpp

```diff
@@ -1211,3 +1187,30 @@ void SILGenFunction::emitIVarInitializer(SILDeclRef ivarInitializer) {
 
   emitEpilog(loc);
 }
+
+
+void SILGenFunction::emitDistributedActorFactory(FuncDecl *fd) {
+  /// NOTE: this will only be reached if the resolve function is actually
+  ///       demanded. For example, by declaring the actor as `public` or
+  ///       having at least one call to the resolve init.
+
+  SILLocation loc = fd; // NOTE: forgot if this is the right one for all locs.
+  ClassDecl *actor = cast<ClassDecl>(fd->getDeclContext()->getAsDecl());
+  assert(actor->isDistributedActor());
+
+  // Step 1: get the uninitialized allocation from the runtime system.
+  auto &ctx = getASTContext();
+  auto builtinName = ctx.getIdentifier(
+      getBuiltinName(BuiltinValueKind::InitializeDistributedRemoteActor));
+  auto returnType = getLoweredType(actor->getInterfaceType());
+  auto *metaTypeInfo = F.getArgument(0);
```

Comment:
Probably the wrong argument number.

Reply:
That'll be the last param, I'll handle that :)